### DISCUSSION TITLE
Copy a pseudo

### DISCUSSION THREAD

**Discussion by DZY-Delta** [kaiserdj/Darkorbit-client#47] [Q&A]
Hi, i'm on Mac and i can't copy my username with the keyboard and right click doesn't work.
My username is "PΉOΣПIX-ΉOŁŁOШ" so I can't write it directly. How can i do ?

**Comment by kaiserdj** [comment]
https://user-images.githubusercontent.com/5487950/138665979-0dc1b0f9-5da2-42ae-a19b-8e4019cdb3bc.mp4

Doesn't that menu come out to you?

**Reply by DZY-Delta** [reply]
it comes but I was talking where you can save many user 
![EA507ED6-4816-4BFD-BB14-03FAE2A8A866](https://user-images.githubusercontent.com/93121845/138669904-a608afbf-05b8-47e8-9d5d-79501c6d0b27.jpeg)

**Comment by kaiserdj** [comment]
aaa, ok now I understand what you mean, I add as a future improvement but while you can use a combination of keys to paste (in windows it is ctrl + v)

**Reply by DZY-Delta** [reply]
Yes, on Mac it’s CMD + V but doesn’t work, and right click neither
